packages/core/src/VNodes/AbstractNode.ts

```diff
@@ -36,6 +36,12 @@ export abstract class AbstractNode {
     toString(): string {
         return this.name;
     }
+    /**
+     * Return the text content of this node.
+     */
+    text(): string {
```

Comment:
😢 Why do you need this ?

Reply:
In the DOM, `Node` has `textContent` which does the same thing so I don't find this shocking.
I need it because a user might not use `CharNode` and there are enough use cases for finding out what the text content of a node is.

Reply:
Well, what you call "enough uses cases", I call it `LinkComponent` line 46 and that's all... ;-) Maybe keep the name `textContent` at least ? Though it would also be a verb-less function. Maybe a getter if we want to stay closer to the dom ?

Reply:
We used to have a `text` field in the description of the `VNode` in DevTools, which I find useful. In many tests we import `CharNode` just so we can do `(node as CharNode).char`... There's plenty of cases in the DOM where I want to use textContent and I don't see why it would be different here.

Reply:
Ok then a `textContent` getter and you update the tests accordingly such that the added feature looks useful ? :)